File: ClassM8Web/m8Src/edu/classm8web/rs/result/EmoteResult.java
package edu.classm8web.rs.result;

import java.util.List;

import edu.classm8web.database.dto.Emote;

public class EmoteResult extends Result {
	private List<Emote> emotes;
	
	public EmoteResult() {}
	
	public List<Emote> getEmotes() {
		return emotes;
	}
	
	public void setEmotes(List<Emote> ids) {
		this.emotes = ids;
	}
	
}

File: ClassM8_Client/ClassM8_Client/Data/EmoteResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace ClassM8_Client.Data
{
    [DataContract]
    class EmoteResult : Result
    {

        [DataMember]
        private List<Emote> emotes;

        public EmoteResult() { }

        public List<Emote> getEmotes()
        {
            return emotes;
        }

        public void setEmotes(List<Emote> ids)
        {
            this.emotes = ids;
        }

    }

}
